masschallenge/impact-api
# [AC-6535] Allocator analysis view: Judge criteria are not displaying remaining capacity

https://masschallenge.atlassian.net/browse/AC-6535

To test:
- load clean-db

- fix gender criteria, which were created wrongly on clean db:
```
from accelerator.models import CriterionOptionSpec
CriterionOptionSpec.objects.filter(option="female").update(option="f")
CriterionOptionSpec.objects.filter(option="male").update(option="m")
```
- start servers, visit http://localhost:1234/allocator and select `2017 BOS CH IL`
- note that you should see "Available" values for roles and genders (except Lawyer, which we actually did run out of in that round)
- hit `http://localhost:8000/api/v1/analyze_judging_round/48/` to see the API bundle returned
